nmdc_schema/migrators/migrator_from_11_8_0_to_11_9_0.py

```diff
@@ -0,0 +1,100 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+
+class Migrator(MigratorBase):
+    r"""Migrates a database between two schemas."""
+
+    _from_version = "11.8.0"
+    _to_version = "11.9.0"
+    wfe_mappings = {}
+    data_objects_mapping = {}
+
+    def upgrade(self) -> None:
+        r"""Migrates the database from conforming to the original schema, to conforming to the new schema.
+        This migrator addresses issue https://github.com/microbiomedata/nmdc-schema/issues/1727.
+        This will deprecate nmdc:MetagenomeSequencing type within the workflow_execution_set collection. Fields from the deprecated records will be migrated to the related DataGeneration records.
```

Comment:
```suggestion
        This will delete nmdc:MetagenomeSequencing records within the workflow_execution_set collection. Fields from the deleted records will be migrated to the related DataGeneration records.
```